README.md

```diff
@@ -178,10 +187,9 @@ If all nodes in the cluster are running Bottlerocket and require the same `updat
 kubectl label node $(kubectl get nodes -o jsonpath='{.items[*].metadata.name}') bottlerocket.aws/updater-interface-version=2.0.0
 ```
 
-If you must support `updater-interface-version` 1.0.0, please [open an issue](https://github.com/bottlerocket-os/bottlerocket-update-operator/issues/new/choose) and tell us about your use case.
-
 ### A Note About Removing Labels
 
+(TODO - is this still true?)
```

Comment:
cc @gthao313 - You had done alot of work in this release for cleanup. Is this chunk still relevant?

Reply:
> you must manually clean up the Custom Resource if you choose to delete only the `updater-interface-version`. The Custom Resource can be deleted like so:
this's out of date. Brupop will automatically garbage collect the Custom Resource `brs`. Therefore, I think we just need remove this that sentence.